### PR TITLE
added vibration feedback post scanning a ticket

### PR DESCRIPTION
**Pull request by mhlangagc**

### CODE REVIEW

File: src/Accounts/eventScanner.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ export default class EventScanner extends Component {
       clearTimeout(this.debounce);
       this.debounce = setTimeout(() => {
         this.setState({read: null})
+        Vibration.vibrate(0.5);
```
Comment (jwietelmann) [inline src/Accounts/eventScanner.js]: You haven't imported the `Vibration` package.

Comment (mhlangagc) [reply src/Accounts/eventScanner.js]: My bad.

**Comment by jwietelmann** [comment]
Trying to figure out why Travis isn't firing...

**Comment by jwietelmann** [comment]
I checked out the branch to test on my phone. There's a pretty nasty lag on this from the time you scan until the vibration kicks in. It also vibrates even when you scan a bad QR. My suggestion is to file an issue and let us deal with the idea more in-depth. I'm going to close this.

**Pull request closed without merging**